openmetadata-docs/content/deployment/kubernetes/values.md

```diff
@@ -9,7 +9,6 @@ This page list all the supported helm values for OpenMetadata Helm Charts.
 
 ## Global Chart Values
 
-<Table>
```

Comment:
Why are the modifications required in this file here ? @pmbrull

Reply:
the table was not rendering properly. The table tag is not required. now it works as expected